### PR TITLE
NXBT-3309: add link to the created PR on related Jira issue(s)

### PR DESCRIPTION
**Pull request by atchertchian**
Created as draft as i need #5 and #7 to be merged first.

Note the feature is also conditioned to a config enablement.

### CODE REVIEW

**Comment by atchertchian** [comment]
https://qa.nuxeo.org/jenkins/job/Misc/job/nuxeo-tools-hooks-branches/view/change-requests/job/PR-8 ran ok

**Comment by atchertchian** [comment]
Rebased according to changes on #7

File: nuxeo-tools-hooks/nxtools/hooks/services/github_service.py [hunk 1]
```diff
@@ -337,6 +337,10 @@ def github_notify(self, pull_request):
         jira = services.get(JiraService)
         keys = jira.get_issue_ids_from_pullrequest(pull_request)
 
+        if len(keys) > 0:
```
Comment (BoboTiG) [inline nuxeo-tools-hooks/nxtools/hooks/services/github_service.py:340]:
```suggestion
        if keys:
```
Minor improvement.

Comment (atchertchian) [reply nuxeo-tools-hooks/nxtools/hooks/services/github_service.py:340]: I actually changed it :) will change it back

**Comment by atchertchian** [comment]
(force-pushed to sign the commits and realign on parent branch feature-NXBT-3308-gh-comment-jira-link)

**Comment by atchertchian** [comment]
Rebased on master after merge of #7

**Pull request merged**